### PR TITLE
Fixed redrawing a skeleton (Shift+N)

### PR DESCRIPTION
**Pull request by bsekachev**
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by bsekachev** [comment]
@klakhov 

Could you please review this patch ASAP?

File: tests/cypress/integration/skeletons_pipeline.js [hunk 1]
```diff
@@ -166,14 +169,39 @@ context('Manipulations with skeletons', () => {
             cy.get(selector).should('not.exist');
         }
 
-        it('Creating and removing a skeleton shape', () => {
+        it.skip('Creating and removing a skeleton shape', () => {
```
Comment (klakhov) [inline tests/cypress/integration/skeletons_pipeline.js:172]: Why skipping? Looks like good basic test.
btw why we also need to `cy.removeAnnotations()` after deleting skeleton, it seems we dont save it anyway

File: tests/cypress/integration/skeletons_pipeline.js [hunk 2]
```diff
@@ -166,14 +169,39 @@ context('Manipulations with skeletons', () => {
...
+            cy.get('#cvat_canvas_shape_1').trigger('mousemove').should('have.class', 'cvat_canvas_shape_activated');
+            cy.get('body').trigger('keydown', { keyCode: 78, code: 'KeyN', shiftKey: true });
+            cy.get('.cvat-canvas-container')
+                .click(skeletonPosition.xtl + REDRAW_MARGIN, skeletonPosition.ytl + REDRAW_MARGIN)
+                .click(skeletonPosition.xbr + REDRAW_MARGIN, skeletonPosition.ybr + REDRAW_MARGIN);
+            cy.get('.cvat-cursor-control').should('have.class', 'cvat-active-canvas-control');
+            cy.get('#cvat_canvas_shape_1').within(() => {
+                cy.get('rect').then(($rect) => {
+                    expect(+$rect[0].getAttribute('x')).to.be.gt(prevX);
+                    expect(+$rect[0].getAttribute('y')).to.be.gt(prevY);
```
Comment (klakhov) [inline tests/cypress/integration/skeletons_pipeline.js:201]: Maybe check for more specific number than just greater? Something like `to be in [prevX+MARGIN/2; prevX+MARGIN*1.5 ]`

File: tests/cypress/integration/skeletons_pipeline.js [hunk 3]
```diff
@@ -184,7 +212,7 @@ context('Manipulations with skeletons', () => {
             cy.removeAnnotations();
         });
 
-        it('Splitting two skeletons and merge them back', () => {
+        it.skip('Splitting two skeletons and merge them back', () => {
```
Comment (klakhov) [inline tests/cypress/integration/skeletons_pipeline.js:215]: Is this test broken? For me it fails locally.
image

**Review by klakhov — COMMENTED** [review]
Redraw feature works nice for me :grinning:

File: tests/cypress/integration/skeletons_pipeline.js (hunk 1 above)
Comment (bsekachev) [reply tests/cypress/integration/skeletons_pipeline.js:172]: Oh, forgot to delete ``skip``

> why we also need to cy.removeAnnotations()

To reset object IDs, and be sure that the next created object will have client ID === 1

File: tests/cypress/integration/skeletons_pipeline.js (hunk 2 above)
Comment (bsekachev) [reply tests/cypress/integration/skeletons_pipeline.js:201]: It is not possible here, there are some translated coordinates (with offset, scale, etc)

File: tests/cypress/integration/skeletons_pipeline.js (hunk 3 above)
Comment (bsekachev) [reply tests/cypress/integration/skeletons_pipeline.js]: This error has been fixed here #4892, I do not have any ideas about why you see it now.
I will run /check command
Locally it passes for me

image

Comment (klakhov) [reply tests/cypress/integration/skeletons_pipeline.js]: Ok, i see CI is green, it seems something on my side

**Review by klakhov — APPROVED** [review]

**Pull request merged**